client/packages/coldchain/src/Monitoring/api/TemperatureBreach/hooks/document/useTemperatureBreachNotifications.ts

```diff
@@ -0,0 +1,14 @@
+import { useTemperatureBreachApi } from '../utils/useTemperatureBreachesApi';
+import { useQuery } from '@openmsupply-client/common';
+import { ListParams } from '../../api';
+
+// Fetches temperature breaches, without raising a permission denied error if one is returned by the API
+export const useTemperatureBreachNotifications = (queryParams: ListParams) => {
```

Comment:
new hook which ignores errors - if the breach api call fails due to permissions, we don't want to notify users that something they didn't ask for has gone wrong